src/main/java/gov/cms/madie/qrda/CqmMeasure.java

```diff
@@ -0,0 +1,34 @@
+package gov.cms.madie.qrda;
+
+import lombok.*;
+
+import java.util.ArrayList;
+import java.util.List;
+
+@Data
+@Builder
+@NoArgsConstructor
+@AllArgsConstructor
+public class CqmMeasure {
```

Comment:
So do you think We can move these models into our package as a part of this story ?

Reply:
If it's okay, I'd rather move them as part of another story. I've already been on this story for well over what was projected. It'd also give us a better opportunity to check them over for our purposes and make sure we package them appropriately.